GitUI/CommandsDialogs/RevisionDiffController.cs

```diff
@@ -74,6 +78,96 @@ public sealed class ContextMenuSelectionInfo
 
     public sealed class RevisionDiffController : IRevisionDiffController
     {
+        private readonly Func<IGitModule> _getModule;
+        private readonly IFullPathResolver _fullPathResolver;
+
+        public RevisionDiffController(Func<IGitModule> getModule, IFullPathResolver fullPathResolver)
+        {
+            _getModule = getModule;
+            _fullPathResolver = fullPathResolver;
+        }
+
+        public void SaveFiles(List<FileStatusItem> files, Func<string, string?> userSelection)
+        {
+            if (files is null)
+            {
+                throw new ArgumentNullException(nameof(files));
+            }
```

Comment:
❔
```suggestion
            files ?? throw new ArgumentNullException(nameof(files));
```

Reply:
In this case we need to use the discard, e.g., `_ = files ?? throw new ArgumentNullException(nameof(files));`. There's no real benefit here.
Once we move to .NET 7+ we'll be able to use `ArgumentNullException.ThrowIfNull`.